roles/mongodb_install/tasks/main.yml

```diff
@@ -4,3 +4,10 @@
   package:
     name: mongodb-org
     state: present
+  when: specific_mongodb_version is not defined
+
+- name: Install MongoDB Packages (Specific version)
+  package:
+    name: "mongodb-org*{{ specific_mongodb_version }}*"
```

Comment:
I suppose the version should be specified by "=". Am I right?

Reply:
For Debian yes but it turns out installing a specific version of mongodb-org still pulls down the latests version anyway (but not on RedHat based systems). I've added some more fixes addressing this as well as some molecule tests for this scenario. Looks good.